src/main/java/org/folio/validate/TextFieldValidator.java

```diff
@@ -1,20 +1,34 @@
 package org.folio.validate;
 
 import java.util.Arrays;
-import java.util.Collections;
 import java.util.List;
 
 import org.apache.commons.lang3.Validate;
-import org.folio.rest.jaxrs.model.CustomField;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
+import org.folio.rest.jaxrs.model.CustomField;
+
 @Component
 public class TextFieldValidator implements CustomFieldValidator {
+
+  private static final String MAXIMUM_LENGTH_MESSAGE = "Maximum length of this text field is %s";
+
+  @Value("${custom.fields.value.textbox.short.length}")
+  private int TEXT_BOX_SHORT_LENGTH;
```

Comment:
use camelCase for private fields, it's a common convention
UPPER_CASE is usually used for constants

Reply:
@dmtkachenko thanks, fixed